Source/ACE/Entity/Container.cs

```diff
@@ -51,14 +52,54 @@ public Container(AceObject aceObject, ObjectGuid guid)
         {
         }
 
+        public virtual void ContainerPlacement(WorldObject inventoryItem)
+        {
+            var removedItem = inventory.Remove(inventoryItem.Guid);
+            var invList = inventory.OrderBy(o => o.Value.Placement);
+            uint place = 0;
+            AceObject saveableCopy;
+            foreach (var keyValuePair in invList)
```

Comment:
When you either put something in your backpack or you just move it around to a new slot - everything in the pack with a placement >= the placement slot you are going in has to move.   Put 4 things in your pack.   They are in slots 0, 1, 2, 3   Now drag item in slot 3 to slot 0.    3 now is set to 0, 0 is set to 1 and 1 is set to 2 and 2 is set to 3.   Not every move ends up with a total inventory shift.   Only drag to the first of the list does that.   If you drag to the middle, only everything past that has to shift.   If you drag to the last open slot, nothing moves.   By default, when you pick something up off the ground if you do not drag it to a slot, it goes to position 0.   There may be a better way to do that.   This works correctly in all cases with the minimum number of moves. (updates).   More than happy to refactor if there is a better way.